### DISCUSSION TITLE
Update the version of sassc in the gemspec

### DISCUSSION THREAD

**Discussion by patrickdavey** [bootstrap-email/bootstrap-email#173] [Ideas]
Hi there,

Firstly I just _love_ this gem, it's really nice, and the integration with Rails makes life really easy!

Anyway, I work on a pretty legacy product, and our version of sassc was at 2.0.1. When I installed this gem it failed because the `load_paths` method doesn't exist.

The fix is simple, just update the `gemspec` to have a dependency of `2.1` for `sassc`. I forked and added a commit here -> https://github.com/patrickdavey/bootstrap-email/commit/c0f86b140654714c6cd2dddb058b76f48b2794e7 . Do you accept PRs into this repository? I'm happy to make this (very simple) PR if you're happy to merge it.

Thanks again for the great gem.

**Comment by stuyam** [comment]
PRs are very much encouraged and supported! That would be great @patrickdavey thank you!
